gbeaudry321/ser401-NASA-psyche-mission
# Us375 probe zone ip gbeaudry

changed ship to be psyche infopoint, added probe infopoint to satellite map area.
 
to test:
Approach ship
press 'E'
check slides
close widget

Use teleport menu to go to satellite zone
approach psyche model
press 'E'
check slides
close widget

Expected behavior:
for ship: Psyche Mission widget opens
for satellite zone: probe info point widget opens
'E' should appear above ship and probe info points. Outline appears for ship but not probe ip.
